### ISSUE TITLE
Provide missing labels and descriptions for buttons

### ISSUE THREAD

**Issue by glowcloud** [swagger-api/swagger-ui#10699]
Provide labels and descriptions for buttons, e.g.:

- global `Authorize` button - provide description for authorized/unauthorized state
- operation-level `Authorize` button - provide indication which operation will be authorized, remove redundant `button` from the label
- `Authorize` button - provide indication that the button will open a dialog
- `X` (close) button in Authorization dialog - provide label
- `Remove array item` button in parameters - provide label
- `Copy to clipboard` button - provide label and indication about what will be copied (e.g. cURL)

**Comment by moskvin** [comment]
I've opened PR #10908 which addresses this. If you're affected by this issue, an upvote or comment there might help it get reviewed. 🙏
